docs/source/releases/v1_10_0_dev0.rst

```diff
@@ -3,10 +3,19 @@ Version 1.10.0.dev0 (2023-04-04)
 
 Enhancements
 ============
+Add a validator for YAML-based plugins.
```

Comment:
@jsolbrig FYI - I added a bulleted summary list right under the version header to CHANGELOG_TEMPLATE.rst.  Just for future reference, I'll add it this time.  I end up referencing the brief summary all the time!

Reply:
Looks good to me. Sorry, I guess I just hadn't edited the changelog for this one yet.